examples/abitest/example.toml

```diff
@@ -0,0 +1,12 @@
+name = "abitest"
+
+[modules]
+module_0 = { Cargo = { cargo_manifest = "examples/abitest/module_0/rust/Cargo.toml" } }
+module_1 = { Cargo = { cargo_manifest = "examples/abitest/module_1/rust/Cargo.toml" } }
+
+[clients]
+cpp = { Bazel = { bazel_target = "//examples/abitest/client:client" }, additional_args = [
+  "--test_exclude=(Storage|GrpcServerServerStreamingMethod)",
+  "--cert_chain=../../../../../../../../examples/certs/local/local.pem",
+  "--private_key=../../../../../../../../examples/certs/local/local.key",
```

Comment:
I know it's a minor thing, but the `../../../../../../../../` paths honestly confuse me so much. But I guess with us moving away from bazel that's a seperate thing. :)

Reply:
Yeah, that's orthogonal to this change, please ignore for now :)

Reply:
Feel free to ignore this since it's not actionable, but purely to satisfy my curiosity,: 

This basically navigates us out of the bazel runner directory and back into the working directory to load the certs, right? Had we considered instead copying the certs in into the bazel runner dir with a build rule?